daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-discover-services.md

```diff
@@ -19,114 +19,94 @@ This article demonstrates how to deploy services each with an unique application
 
 Dapr allows you to assign a global, unique ID for your app. This ID encapsulates the state for your application, regardless of the number of instances it may have.
 
-{{< tabs Dotnet Java Python Go JavaScript Kubernetes>}}
+{{< tabs Python JavaScript ".NET" Java Go Kubernetes >}}
 
 {{% codetab %}}
 
 ```bash
+dapr run  --app-id checkout --app-protocol http --dapr-http-port 3500 -- python3 checkout/app.py
 
-dapr run --app-id checkout --app-port 6002 --dapr-http-port 3602 --dapr-grpc-port 60002 dotnet run
-
-dapr run --app-id orderprocessing --app-port 6001 --dapr-http-port 3601 --dapr-grpc-port 60001 dotnet run
-
+dapr run --app-port 8001 --app-id order-processor --app-protocol http --dapr-http-port 3501 -- python3 order-processor/app.py
```

Comment:
In all of these --app-id should come first. (like you have for the checkout app)
```suggestion
dapr run --app-id order-processor --app-port 8001  --app-protocol http --dapr-http-port 3501 -- python3 order-processor/app.py
```